README.md

```diff
@@ -20,6 +21,15 @@ in training set).
 7. Fish body must not be folded and should have no curvature.
 
 These do not need to be adhered to if properly set up/modified for a specific use case.
+The model is available on data commons osc as "Drexel_metadata_generator" at https://datacommons.tdai.osu.edu/dataverse/fish-traits/ 
+However this model is not published yet. To download you need an acount on datacommons.tdai.osu and you will need to create a DATAVERS_API_TOKEN 
```

Comment:
```suggestion
However this model is not published yet. To download you will need an account on datacommons.tdai.osu and you will need to create an API Token and set the  DATAVERSE_API_TOKEN environment variable.
```

Reply:
We should clarify the following things first:
1. Can the model be made public at this point? If yes, we should make it so, and also consider putting it on [Huggingface](https://huggingface.co/imageomics). If not, what are the criteria for making it public.
2. If the model can't be public yet, the description above won't cut it, because people will need not only an account put also be given access.

Reply:
1- make the model public, i say yes but that is not my call, the ball is in Jane Greenberg (@hlapp, you were cc in the email, the response from Jane was unclear). for huggingface, yes that is next on my todo list. 

2- Sure but keep it mind that right now, the point (in priority) is to make this project available and documented for people from our institution BGNN/imageomics. So I assume that people will have (or should have) access to the datacommons/dataverse.